3.3/.sti/bin/assemble

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /opt/openshift/etc/generate_container_user
```

Comment:
@csrwng thanks, I will remove it from assemble (it seems to be harmless as the test is passing)